test/ParserTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace CSVParser\Tests;
+
+use CSVParser\Parser;
+
+final class ParserTests extends \PHPUnit_Framework_TestCase
+{
+    protected $file;
+
+    public function setUp()
+    {
+        $this->file = tmpfile();
+    }
+
+    public function tearDown()
+    {
+        fclose($this->file);
+    }
+
+    /*
+     * @test
+     */
+    public function testConstruction()
+    {
```

Comment:
Where are the unit tests for all of the edge cases of constructing a Parser object?

Reply:
I have not reached a point in which I know how to do the test.  I was first trying to make sure that an object was able to be made and using the correct terms and system.

Reply:
from command line you'll use `./vendor/bin/phpunit test`